recipes/mir_eval/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mir_eval" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f87310e326ad9f80ba65e3f0d1ced2013a2845016fc51d7af56c79195668d216
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install .  -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - future
+    - numpy >=1.7.0
+    - python
+    - scipy >=1.0.0
+    - six
+
+test:
+  imports:
+    - mir_eval
+
+about:
+  home: https://github.com/craffel/mir_eval
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "Common metrics for common audio/music processing tasks."
+  doc_url: http://labrosa.ee.columbia.edu/mir_eval/
```

Comment:
I think the `doc_url` should be http://craffel.github.io/mir_eval/ instead. Other than this, the recipe LGTM.